acl_loader/main.py

```diff
@@ -706,7 +706,10 @@ def show_table(self, table_name):
                 if not val["ports"]:
                     data.append([key, val["type"], "", val["policy_desc"], stage])
                 else:
-                    ports = natsorted(val["ports"])
+                    if isinstance(val['ports'], str):
```

Comment:
Have you tried the case where "ports" is a comma-separated string like "Ethernet0,Ethernet1,..."? Does this come back as a string or a list?

Reply:
I've tried different cases and the comma separated string is returned as a string, so it shouldn't work and needs to be split.
```
"2": {
	"policy_desc": "Mirror Session",
	"ports": "Ethernet16,Ethernet20", <= return str 
	"stage": "INGRESS",
	"type": "MIRROR"
},
"3": {
	"policy_desc": "Mirror Session",
	"ports": [			  <= return list 
		"Ethernet16",
		"Ethernet20"
	],
	"stage": "INGRESS",
	"type": "MIRROR"
}
```
```
# show acl table
Name       Type    Binding                Description     Stage
---------  ------  ---------------------  --------------  -------
2          MIRROR  Ethernet16,Ethernet20  Mirror Session  ingress

3          MIRROR  Ethernet16             Mirror Session  ingress
                   Ethernet20

```

Reply:
@daall
I`ve tried cases with different acl command and they all work fine:

```
#sudo config acl add table test_1 L3 -p Ethernet0,Ethernet4 -s ingress
"test_1": {
	"policy_desc": "Mirror Session",
	"ports": [
		"Ethernet0",
		"Ethernet4"
	],
	"stage": "INGRESS",
	"type": "MIRROR"
}

#sudo config acl add table test_2 L3 -p "Ethernet0,Ethernet4" -s ingress
"test_2": {
	"policy_desc": "Mirror Session",
	"ports": [
		"Ethernet0",
		"Ethernet4"
	],
	"stage": "INGRESS",
	"type": "MIRROR"
}

#sudo config acl add table test_single L3 -p Ethernet0 -s ingress
"test_single": {
	"policy_desc": "Mirror Session",
	"ports": [
		"Ethernet0"
	],
	"stage": "INGRESS",
	"type": "MIRROR"
}
```

This bug happens only when I manually write the wrong value in config db.json, as this stated in issue(instead of list, write single str port)
```
"1": {
	"stage": "INGRESS",
	"type": "MIRROR",
	"policy_desc": "Mirror Session",
	"ports": "Ethernet16" <= incorrect value
}
```